src/features/shareOffer/helpers/getOfferUrl.test.ts

```diff
@@ -0,0 +1,13 @@
+import { mockOffer } from 'features/bookOffer/fixtures/offer'
+import { WEBAPP_V2_URL } from 'libs/environment'
+import { renderHook } from 'tests/utils'
+
+import { getOfferUrl } from './getOfferUrl'
+
+describe('getOfferUrl', () => {
+  it('should return the url with the correct path and offer id ', () => {
+    const { result } = renderHook(() => getOfferUrl(mockOffer.id))
+
+    expect(result.current).toEqual(`${WEBAPP_V2_URL}/offre/146112`)
```

Comment:
Vu que c'est pas un hook tu peux direct faire ça :
```suggestion
    const offerUrl = getOfferUrl(mockOffer.id)

    expect(offerUrl).toEqual(`${WEBAPP_V2_URL}/offre/146112`)
```